blueflood-core/src/main/java/com/rackspacecloud/blueflood/cache/TenantTtlProvider.java

```diff
@@ -26,4 +26,6 @@
     public void setTTL(String tenantId, Granularity gran, RollupType rollupType, TimeValue ttlValue) throws Exception;
 
     public TimeValue getTTLForStrings(String tenantId) throws Exception;
+
+    public TimeValue getConfigTTLForIngestion() throws Exception;
```

Comment:
Maybe I misunderstood, but I thought @chinmay-gupte said to expose the getFinal TTL method in the interface, instead of this one?